Python/optimizer.c

```diff
@@ -705,7 +705,11 @@ translate_bytecode_to_trace(
                                 // The number of caches is smuggled in via offset:
```

Comment:
Oh I just realized that we don't need `OPARG_SET_IP` any more -- it's not used at all! We can renumber `OPARG_SAVE_CURRENT_IP` to 7, and rename it to `OPARG_SET_RETURN_OFFSET` while we're at it.